src/toast/interactive/utils.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) 2024-2025 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+"""Utilities useful in interactive sessions."""
+
+import os
+
+import numpy as np
+
+from ..mpi import Comm
+
+
+def create_outdir(mpicomm, topdir=None, subdir=None):
+    """Create the top level output directory and per-test subdir.
+
+    If `topdir` is not specified, a directory is created in the current
+    working directory.
+
+    Args:
+        mpicomm (MPI.Comm): the MPI communicator (or None).
+        topdir (str): the top-level directory to create.
+        subdir (str): the sub directory for this test.
+
+    Returns:
+        str: full path to the subdir if specified, else the topdir.
+
+    """
+    if topdir is None:
+        pwd = os.path.abspath(".")
+        topdir = os.path.join(pwd, "toast_output")
+    retdir = topdir
+    if subdir is not None:
+        retdir = os.path.join(topdir, subdir)
+    if (mpicomm is None) or (mpicomm.rank == 0):
+        if not os.path.isdir(topdir):
```

Comment:
Matter of preference, but you don't need to check for existence with `os.makedirs(topdir, exist_ok=True)`.

Reply:
Good point, will fix it in the next PR.